### PR TITLE
Help debug stalled txs

### PR DESCRIPTION
**Pull request by danfinlay**
Added a few stylistic changes to the tx list to make it easier to debug users' stalled txs:
- Show nonce in tx list
- Add tooltip to pending tx icon (people have been confused by it) for until we have a better icon.

screen shot 2017-05-21 at 2 14 56 pm
screen shot 2017-05-21 at 2 14 50 pm

### CODE REVIEW

File: ui/app/components/transaction-list-item.js [hunk 1]
```diff
@@ -39,6 +40,8 @@ TransactionListItem.prototype.render = function () {
     txParams = transaction.msgParams
   }
 
+  const nonce = (new BN(txParams.nonce.substr(2))).toString(10)
```
Comment (kumavis) [inline ui/app/components/transaction-list-item.js]: `txParams.nonce` may be a Number

**Review by kumavis — CHANGES_REQUESTED** [review]

File: ui/app/components/transaction-list-item.js (hunk 1 above)
Comment (kumavis) [inline ui/app/components/transaction-list-item.js]: `web3.eth.sendTransaction({ from: web3.eth.accounts[0], to: web3.eth.accounts[0], none: 10 })`

Comment (kumavis) [inline ui/app/components/transaction-list-item.js]: web3.currentProvider.sendAsync([ id: 1, method: 'eth_sendTransaction', params: [{
  from: web3.eth.accounts[0],
  to: web3.eth.accounts[0],
  nonce: 10
}])

**Comment by flyswatter** [comment]
Fixes #1464.

**Review by kumavis — APPROVED** [review]

**Pull request merged**